tests/event/select.rs

```diff
@@ -277,6 +278,7 @@ fn test_select_with_sockets() {
 #[test]
 #[serial] // for `setrlimit` usage, and `crate::init`
 fn test_select_with_maxfd_sockets() {
+    use rustix::fd::{FromRawFd as _, OwnedFd};
```

Comment:
I'm curious why these lines were needed in the tests, that weren't otherwise changed.